test/uk/gov/hmrc/tai/service/CodingComponentServiceSpec.scala

```diff
@@ -52,57 +53,23 @@ class CodingComponentServiceSpec extends PlaySpec with MockitoSugar with FakeTai
         val service = createSut
         val incomeType: CodingComponent = CodingComponent(BalancingCharge, None, 0, "BalancingCharge Description")
         when(taxAccountConnector.codingComponents(any(), any())(any()))
-          .thenReturn(Future.successful(TaiSuccessResponseWithPayload(codingComponents :+ incomeType)))
+          .thenReturn(Future.successful(codingComponents :+ incomeType))
 
         val result = service.taxFreeAmountComponents(generateNino, currentTaxYear)
         Await.result(result, 5 seconds) mustBe codingComponents
       }
     }
 
     "return empty CodingComponents" when {
-      "error is TaiNotFoundResponse" in {
+      "error is NotFoundException at Connector level" in {
         val service = createSut
         when(taxAccountConnector.codingComponents(any(), any())(any()))
-          .thenReturn(Future.successful(TaiNotFoundResponse("no coding components found")))
+          .thenReturn(Future.successful(Seq.empty[CodingComponent]))
 
         val result = service.taxFreeAmountComponents(generateNino, currentTaxYear)
         Await.result(result, 5 seconds) mustBe Seq.empty
       }
     }
-
-    "return Tai Failure response" when {
```

Comment:
why are these tests deleted and not updated?

Reply:
Tests for exceptions thrown by the service. The service's logic has been removed and is now just returning whatever the connector returns so not needed to test in serviceSpec, only connectorSpec.